### DISCUSSION TITLE
Postgres Password incorrect

### DISCUSSION THREAD

**Discussion by iotalol** [plausible/analytics#2546] [Self-Hosted Support] [answered]
(Postgrex.Error) FATAL 28P01 (invalid_password) password authentication failed for user "postgres"
Steps to try and fix:
Changing password in postgre itself by sudo docker exec -it hosting_plausible_db_1 bash

**Comment by ruslandoga** [comment, marked as answer]
@iotalol 👋

If you have a different password set for your Postgres instance, you can use [`DATABASE_URL`](https://github.com/plausible/analytics/blob/10a976791261af9960601ab7b3d96ca92d9a0dd0/config/runtime.exs#L53-L58) env var with something like `postgres://your_db_user:your_password@your_db_host:5432/plausible_dev`

More info on https://plausible.io/docs/self-hosting-configuration#database

**Reply by iotalol** [reply]
Holy shit, I've been trying to figure out how to fix this for so long!! Thank you so much :))
